test/lib/usd/translators/testUsdExportFileFormat.py

```diff
@@ -0,0 +1,143 @@
+#!/pxrpythonsubst

```

Comment:
This test is the one new change from the original PR.  All this does is export a poly sphere from Maya using the 'file' command as well as the mayaUSDExport command, checking the default option (which should be usdc) as well as explicitly setting the .usd format to usda and usdc.